lib/backends/webgl/glsl-coordinate-lib.ts

```diff
@@ -264,8 +264,9 @@ export class CoordsGlslLib extends GlslLib {
    */
   protected getOutputPacked3DCoords(shape: [number, number, number], texShape: [number, number]): GlslLibRoutine {
     const packedTexShape = [texShape[0], texShape[1]];
-    const texelsInLogicalRow = shape[2];
-    const texelsInBatch = texelsInLogicalRow * shape[1];
+    const texelsInLogicalRow = Math.ceil(shape[2] / 2);
```

Comment:
After so many tests, I thought we had a pretty good coverage on this library... :0

Reply:
I know.. this was really subtle, only triggered by some specific tensor/tex shapes.